app/Models/ImageMention.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\Model;
+use App\Models\Concerns\SortableTrait;
+use Illuminate\Support\Arr;
+
+/**
+ * Class ImageMention
+ * @package App\Models
+ *
+ * @property integer $image_id
+ * @property integer $post_id
+ * @property integer $entity_id
+ * @property Entity $entity
+ * @property Post $post
+ * @property Entity $target
+ *
+ * @method static self|Builder prepareCount()
+ */
+class ImageMention extends Model
+{
+    use SortableTrait;
+
+    public $fillable = [
+        'entity_id',
+        'image_id',
+        'post_id',
+    ];
+
+    protected $sortable = [
+        'name',
+        'type',
+    ];
+
+    /**
+     * @return \Illuminate\Database\Eloquent\Relations\BelongsTo
+     */
+    public function target()
+    {
+        return $this->belongsTo('App\Models\Images', 'image_id', 'id');
```

Comment:
I don't see a model called `Images` (plural). Does this function get called? If it never gets called, we can remove it altogether.